design-system/react/src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,33 @@
+import { styled } from "@fuel-ui/css";
+
+import { createComponent } from "../../utils";
+
+export type LogoProps = {
+  size?: number;
+};
+
+const Svg = styled("svg");
+
+export const Logo = createComponent<LogoProps>(
+  ({ size = 60, className, ...props }) => {
+    return (
+      <Svg
```

Comment:
I think the SVG should be a param, as the component is a genric Logo, if the intent is to be the Fuel logo, maybe we should rename to `FuelLogo`

Reply:
The intent here is to be the FuelLogo, of course. But since this is our design system, we really think we need to explicitly say that is `FuelLogo` @luizstacio ?

Reply:
In my opinion yes. As a fuel-UI user, when using Logo, I would expect it to be an avatar-type component and not a fuel logo.

Reply:
✅